libs/stats/docs/pq-cfg.yaml

```diff
@@ -0,0 +1,14 @@
+# Connfiguration file for `odc-stats run` command
+
+# Configure Plugin and Product Attributes
+plugin: pq
+
+# Generic product attributes
+product:
+  name: pc_s2_annual
+  short_name: pc_s2_annual
+  version: 1.0.0
+  product_family: pixel_quality_statistics
+
+  collections_site: explorer.digitalearth.africa
+  producer: digitalearthafrica.org
```

Comment:
With index changes we now need

```yaml
  region_code_format: "x{x:03d}y{y:03d}"
```
added to `.product`, I'd put it after `producer` it's "fixed for Africa configs" kinda setting.